docs/quick-ingestion-guides/redshift/configuration.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Configuration
+---
+# Configuring Your Redshift Connector to DataHub
+
+Now that you have created a DataHub user in Redshift in [the prior step](setup.md), it's now time to set up a connection via the DataHub UI.
+
+## Configure Secrets
+
+1. Within DataHub, navigate to the **Ingestion** tab in the top, right corner of your screen
+
+<p align="center">
+  <img width="75%" alt="Navigate to the &quot;Ingestion Tab&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/redshift/redshift_ingestion_ingestion_button.png"/>
+</p>
+
+:::note
+If you do not see the Ingestion tab, please contact your DataHub admin to grant you the correct permissions
+:::
+
+2. Navigate to the **Secrets** tab and click **Create new secret**
+
+<p align="center">
+   <img width="75%" alt="Secrets Tab" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/redshift/redshift_ingestion_secrets_tab.png"/>
+</p>
+
+3. Create a Redshift User's Password secret
+
+This will securely store your Redshift User's password within DataHub
+
+* Click **Create new secret** again
+* Enter a name like `REDSHIFT_PASSWORD` - we will use this later to refer to the secret
+* Enter your `datahub` redshift user's password
+* Optionally add a description
+* Click **Create**
+
+<p align="center">
+   <img width="75%" alt="Redshift Password Secret" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/redshift/redshift_ingestion_password_secret.png"/>
+</p>
+
+## Configure Recipe
+
+5. Navigate to the **Sources** tab and click **Create new source**
```

Comment:
This should not be step 5, right?

Reply:
Since it's in a separate section should be new list.

Here is an idea about how to make it more concise:


Click on the **Sources** tab and then **Create new source**.

Select Tableau.

Fill in the Tableau Recipe form:

a. **Host URL**: URL of your Tableau instance (e.g., https://15az.online.tableau.com/).

b. **Username**: Use the TABLEAU_USERNAME secret (e.g., "${TABLEAU_USERNAME}").

c. **Password**: Use the TABLEAU_PASSWORD secret (e.g., "${TABLEAU_PASSWORD}").

To filter for specific site and project, use the site and project_pattern fields. Click **Next** when done.

Reply:
Should be a new set of steps, or is that we are doing?

Reply:
in other docs